### ISSUE TITLE
Missing functions

### ISSUE THREAD

**Issue by oscar-broman** [samp-incognito/samp-gvar-plugin#1]
These definitions don't exist in the code:
https://github.com/samp-incognito/samp-gvar-plugin/blob/master/src/main.cpp#L359-L361

**Comment by samp-incognito** [comment]
Good point. Those were features I meant to add eventually. Maybe I'll get around to it some day...

**Comment by oscar-broman** [comment]
That'd be great. Currently, iterating through keys using the upper index appears to be `O(log(N!))`. It can't even handle ~1k keys very well.
